config/quality/findbugs/findbugs-filter.xml

```diff
@@ -8,13 +8,11 @@
     <Match>
         <Class name="~.*\.Manifest\$.*"/>
     </Match>
-    <!-- All bugs in test classes, except for JUnit-specific bugs -->
     <Match>
         <Class name="~.*\.*Test" />
-        <Not>
-            <Bug code="IJU" />
-        </Not>
     </Match>
-
-
+    <!-- TODO Should be removed after #137 is merged -->
```

Comment:
I don't think that you want to exclude classes from findbugs generally. We can always include those classes when that's really needed in the future